src/_includes/banner.html

```diff
@@ -4,6 +4,8 @@
   <div class="site-banner site-banner--default" role="alert">
     {% comment %} For headings use <h3> elements. {% endcomment -%}
     <a href="https://g.co/flutterday">Flutter Day is June 25th. Register now!</a><br>
+    Tweet your questions for #AskFlutter. For more info,
+    see the <a href="https://twitter.com/FlutterDev/status/1272544935426678785">tweet!</a>.
```

Comment:
There's an extra `.`, this is kind of long, and the link is unclear. How about maybe:

```suggestion
    Tweet your questions for #AskFlutter.
    <a href="https://twitter.com/FlutterDev/status/1272544935426678785">Details here!</a>
```

Reply:
Also... Is there a good reason to keep the commented-out text?